uk.co.compucorp.civicrm.hrleaveandabsences/tests/phpunit/CRM/HRLeaveAndAbsences/BAO/LeaveBalanceChangeTest.php

```diff
@@ -3675,6 +3659,7 @@ private function getBalanceChangesForPeriodEntitlement($leavePeriodEntitlement)
 
   private function calculateAmountForDate(LeaveRequest $leaveRequest, DateTime $date, $amount) {
```

Comment:
Now that the `LeaveBalanceChange::calculateAmountForDate()` method barely has any logic in it, I was expecting to see some more tests deleted/updated. To be honest, I think that for this method we only need these tests:

- One test that returns 0 when `ContactWorkPatternService.getContactWorkDayForDate` returns null
- One test that returns the correct deduction when  `ContactWorkPatternService.getContactWorkDayForDate` is not null